meerk40t/core/elements.py

```diff
@@ -461,14 +461,11 @@ def add(self, data_object=None, type=None, label=None, pos=None):
         node.notify_attached(node, pos=pos)
         return node
 
-    def label_from_source_cascade(self) -> str:
+    def label_from_source_cascade(self, element) -> str:
```

Comment:
I think this will fall into the crashing again. element can sometimes be a operation and that needs to run correctly.